pkg/util/cmp/cmp.go

```diff
@@ -1,20 +1,20 @@
 package cmp
 
 import (
+	"crypto/rsa"
 	"crypto/x509"
 
 	gocmp "github.com/google/go-cmp/cmp"
 )
 
 // Diff is a wrapper for github.com/google/go-cmp/cmp.Diff with extra options
 func Diff(x, y interface{}, opts ...gocmp.Option) string {
-	newOpts := append(
-		opts,
-		// FIXME: Remove x509CertComparer after upgrading to a Go version that includes https://github.com/golang/go/issues/28743
-		gocmp.Comparer(x509CertComparer),
-	)
+	// FIXME: Remove x509CertComparer after upgrading to a Go version that includes https://github.com/golang/go/issues/28743
+	opts = append(opts, gocmp.Comparer(x509CertComparer))
 
-	return gocmp.Diff(x, y, newOpts...)
+	opts = append(opts, gocmp.Comparer(rsaPrivateKeyComparer))
+
+	return gocmp.Diff(x, y, opts...)
```

Comment:
why is this file being changed in 7.1?

Reply:
gocmp panics when you run it on full structure because of ssh keys. It was an issue when I moved logic around. I think it is no-op now. Its a bug in current our Diff logic in master too (see my PLS PR).

Will remove.